Classes/Utility/DatabaseUtility.php

```diff
@@ -25,7 +25,7 @@ class DatabaseUtility
     public static function getRecord(string $table, int $uid, array $fields = ['*'])
     {
         // Ensure we have a valid uid (not 0 and not NEWxxxx) and a valid TCA
-        if (!empty($GLOBALS['TCA'][$table])) {
+        if (($GLOBALS['TCA'][$table] ?? []) !== []) {
```

Comment:
Same here why not use is_array?

Reply:
This is a bit of shorthand. I can't just check for array, but have to check if it's set to avoid an error. I could do `isset($GLOBALS['TCA'][$table] ?? [])`. The explicit way would be to write `isset($GLOBALS['TCA'][$table]) && is_array($GLOBALS['TCA'][$table])`